src/Security/VoterV2/RelayVoter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Security\VoterV2;
+
+use App\Entity\Centre;
+use App\Entity\User;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+
+class RelayVoter extends Voter
+{
+    public const MANAGE_PRO = 'MANAGE_PRO';
+    public const MANAGE_BENEFICIARIES = 'MANAGE_BENEFICIARIES';
+
```

Comment:
I think there is something weird with the permissions here. We have something very similar called MANAGE in Beneficiary and Pro Voters.